domain/src/test/java/org/oppia/android/domain/platformparameter/PlatformParameterModuleTest.kt

```diff
@@ -149,6 +176,12 @@ class PlatformParameterModuleTest {
     fun providePlatformParameterSingleton(
       platformParameterSingletonImpl: PlatformParameterSingletonImpl
     ): PlatformParameterSingleton = platformParameterSingletonImpl
+
+    @Provides
+    @Singleton
```

Comment:
Doesn't need to be scoped--you can drop this.

Ditto for the other module changes in this PR. 'Singleton' is only needed if you have data that needs to be retained (otherwise bindings are recreated upon injection).